requirements_dev.txt

```
@@ -8,7 +8,7 @@ flake8-black>=0.2.4
 doc8
 Pygments
 # testing
-coverage==6.2
+coverage!=6.3
```

Can we change this to `coverage>=6.4`? Basically equivalent, and looks more natural. If we're going to leave it as `!=6.3`, I would suggest adding a comment linking to the issue that we're avoiding by doing so.

---

Guess we could as only really needed for CI. Was trying to avoid being so strict.

```suggestion
coverage>=6.4
```